src/v3/src/types/schema.ts

```diff
@@ -515,6 +515,17 @@ export interface ImageWithTextElement extends UISchemaElement {
   };
 }
 
+export interface ImageLinkElement extends UISchemaElement {
+  type: 'ImageLink';
+  options: {
```

Comment:
You can add width and height properties here to be able to pass (the `fit-content`) from the transformer.

Reply:
@vakaevich-okta I made a commit off your branch with some changes to get rid of the use of the SVG style within the Widget/style.css file. Please take a look and let me know if you have questions. 

https://github.com/okta/okta-signin-widget/commit/42e83097b87a66dcd2384be91c08aca4831633f8

I was also able to fix the issue where you were forced to add ariaLabel to the link instead of just the image. Take a look there as well. I ran against Axe devtools locally and it passes without the ariaLabel on the link. 
![image](https://github.com/okta/okta-signin-widget/assets/97472729/3ee1179a-d53b-41be-b4cd-266600902d03)

Reply:
thanks @glenfannin-okta ! all 4 svg icons look correct?

Reply:
@vakaevich-okta Yep 🙏 I checked all 4 icons and they all look correct after that update.